R/verb-expand.R

```diff
@@ -0,0 +1,44 @@
+#' @importFrom tidyr expand
+#' @export
+expand.tbl_lazy <- function(data, ..., .name_repair = "check_unique") {
+  # TODO sort output?
```

Comment:
I don't think so; I think the usual principle is to respect database behaviour unless the user specifically requests otherwise